src/scaffoldmaker/meshtypes/meshtype_3d_wholebody1.py

```diff
@@ -239,6 +240,28 @@ def generateBaseMesh(region, options):
                                  cylinderShape=cylinderShape,
                                  cylinderCentralPath=cylinderCentralPath, useCrossDerivatives=False)
 
+        # body_coordinates
+        body_coordinates = findOrCreateFieldCoordinates(fieldmodule, name="body_coordinates")
```

Comment:
Rename the variable bodyCoordinates for consistency with other variables as body_coordinates looks funny in later code.
The tmp_ prefix in the following code can remain, don't care.